edenai_apis/apis/nyckel/info.json

```diff
@@ -0,0 +1,24 @@
+{
+    "image": {
+        "search": {
+            "upload_image": {
+                "version": "v1.0.0"
```

Comment:
I'm not sure what this version is intended to refer to.

Reply:
I's supposed to be the api/sdk version used, for your case I guess it would be 0.9 for the 'search' endpoint and 1.0 for the rest